components/Panel/VotePanel/VotePanel.tsx

```diff
@@ -56,11 +77,7 @@ export function VotePanel({ content }: Props) {
         isGovernance={isGovernance}
         voteNumber={resolvedPriceRequestIndex}
       />
-      {hasResults ? (
```

Comment:
this is the source of the bug where the discussion tab didn't show on active votes. active votes don't have results, so we'd never show the tabs